src/components/AlreadyElectrifiedChart.js

```diff
@@ -0,0 +1,23 @@
+import React from 'react'
+
+const electrifiedColor = "rgb(163, 215, 164)"
+const fossilColor = "rgb(255, 87, 34)"
+const AlreadyElectrifiedChart = ({ label, electrifiedPct, fossilPct }) => { //otherPct
+  const formattedData = {
```

Comment:
GitHub fairly points out this variable is created but never used

Reply:
Fair indeed! Resolved in 35f13fa